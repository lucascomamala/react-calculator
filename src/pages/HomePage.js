import React from "react"

const HomePage = () => {
  return (
    <div className="home">
      <h2>Welcome to our page!</h2>
      <p>An electronic calculator is typically a portable electronic device used to perform calculations, ranging from basic arithmetic to complex mathematics.

        The first solid-state electronic calculator was created in the early 1960s. Pocket-sized devices became available in the 1970s, especially after the Intel 4004, the first microprocessor, was developed by Intel for the Japanese calculator company Busicom.

        Modern electronic calculators vary from cheap, give-away, credit-card-sized models to sturdy desktop models with built-in printers. They became popular in the mid-1970s as the incorporation of integrated circuits reduced their size and cost. By the end of that decade, prices had dropped to the point where a basic calculator was affordable to most and they became common in schools.</p>
    </div>
  );
}
export default HomePage
