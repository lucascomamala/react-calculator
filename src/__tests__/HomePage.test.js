import React from 'react';
import renderer from 'react-test-renderer';
import HomePage from '../pages/HomePage';

it('renders correctly', () => {
  const tree = renderer.create(<HomePage />).toJSON();
  expect(tree).toMatchSnapshot();
});
