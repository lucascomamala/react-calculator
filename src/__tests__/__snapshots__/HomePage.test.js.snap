// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <h2>
    Welcome to our page!
  </h2>,
  <p>
    hello from home page
  </p>,
]
`;
